src/main/java/org/alliancegenome/curation_api/jobs/BulkLoadJobExecutor.java

```diff
@@ -9,192 +9,43 @@
 import static org.alliancegenome.curation_api.enums.BackendBulkLoadType.GENE;
 import static org.alliancegenome.curation_api.enums.BackendBulkLoadType.GENE_DISEASE_ANNOTATION;
 
-import java.io.FileInputStream;
-import java.time.OffsetDateTime;
 import java.util.List;
-import java.util.Map;
-import java.util.Map.Entry;
-import java.util.zip.GZIPInputStream;
 
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
 
 import org.alliancegenome.curation_api.dao.loads.BulkLoadFileDAO;
 import org.alliancegenome.curation_api.enums.BackendBulkLoadType;
-import org.alliancegenome.curation_api.enums.OntologyBulkLoadType;
 import org.alliancegenome.curation_api.jobs.executors.AgmDiseaseAnnotationExecutor;
 import org.alliancegenome.curation_api.jobs.executors.AgmExecutor;
 import org.alliancegenome.curation_api.jobs.executors.AlleleDiseaseAnnotationExecutor;
 import org.alliancegenome.curation_api.jobs.executors.AlleleExecutor;
 import org.alliancegenome.curation_api.jobs.executors.GeneDiseaseAnnotationExecutor;
 import org.alliancegenome.curation_api.jobs.executors.GeneExecutor;
 import org.alliancegenome.curation_api.jobs.executors.MoleculeExecutor;
+import org.alliancegenome.curation_api.jobs.executors.OntologyExecutor;
```

Comment:
@markquintontulloch I moved all the ontology stuff out of this file into its own OntologyExecutor it makes this file much smaller :)